### DISCUSSION TITLE
Status of `libunwind-setjmp` library?

### DISCUSSION THREAD

**Discussion by tkchia** [libunwind/libunwind#632] [Q&A]
Good day,

May I know the current development status of the `libunwind-setjmp` library?

It seems to me that &mdash; at least according to [`src/Makefile.am`](https://github.com/libunwind/libunwind/blob/64dc936cfbc9935313341bd8edcc20881bcba529/src/Makefile.am) &mdash; only the IA-64 architecture has an implementation of a fast `setjmp ( )`.  And in particular, on x86-64 Linux with `glibc`, it seems the library pretty much defers to the C library, for both `setjmp ( )` and `longjmp ( )`.

Thank you!

**Comment by bregma** [comment]
If you're targeting GNU/Linux, which already has a fast setjmp/longjmp implementation, why are you trying to use the libunwind implementation? It's really only makes sense for targets that do not already have a native implementation.

I think the comments in the code are pretty self-explanatory as to why the GNU/Linux target is special-cased.
